war/src/main/webapp/scripts/hudson-behavior.js

```diff
@@ -728,20 +728,6 @@ function expandButton(e) {
     layoutUpdateCallback.call();
 }
 
-function inputHasDefaultTextOnFocus() {
```

Comment:
Did you make sure this function is not used elsewhere across the https://github.com/jenkinsci organization?

Reply:
not used https://github.com/search?q=org%3Ajenkinsci+inputHasDefaultTextOnFocus&type=Code